packages/conf-diffutils/conf-diffutils.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+homepage: "https://www.gnu.org/software/diffutils/"
+bug-reports: "chetsky@gmail.com"
+maintainer: "chetsky@gmail.com"
+authors: "GNU Project"
+license: "GPL-3.0-or-later"
+build: [["sh" "-exc" "find . -name ."]]
```

Comment:
```suggestion
build: ["diff" "--help"]
```

Reply:
arrgh, that was stupid!  Thank you for catching it!